packages/client/.eslintrc.cjs

```diff
@@ -13,7 +13,7 @@ module.exports = {
   rules: {
     'simple-import-sort/imports': 'error',
     'simple-import-sort/exports': 'error',
-    'prettier/prettier': 'error',
+    'prettier/prettier': 'off',
```

Comment:
do we really want to turn this off? Are the line endings still causing issues?

Reply:
you can try adding the .gitattributes file to the root dir, similar to the newbie projects